### ISSUE TITLE
java.lang.IllegalStateException: Fragment already added: MainFragment

### ISSUE THREAD

**Issue by semanticer** [toggl/mobileapp#6044]
## 💥 Crash Report
This type of crash generally happens when we try to add the same fragment twice. The most common issue is with the DialogFragment being added twice before dismissing.

It looks like this event plays some major role: 
```
ReportsSuccess
Source:“ShortcutToday”
LoadingTime:“1000”
ProjectsNotSynced:“0”
TotalDays:“7”
```
### 👥 Impact
This impacts around 0.08% of android 2.4.1 users

### 🔗 Context
??

### 🛰 Link
https://appcenter.ms/orgs/Toggl/apps/Toggl-Android/crashes/errors/1324550673u/overview

### 📝  Exception & Stack Trace
`java.lang.IllegalStateException: Fragment already added: MainFragment`
```cs
java.lang.IllegalStateException: Fragment already added: MainFragment{123f7b5 #1 id=0x7f0d017e}
  at android.support.v4.app.FragmentManagerImpl.addFragment(FragmentManager.java:1916)
  at android.support.v4.app.BackStackRecord.executeOps(BackStackRecord.java:765)
  at android.support.v4.app.FragmentManagerImpl.executeOps(FragmentManager.java:2625)
  at android.support.v4.app.FragmentManagerImpl.executeOpsTogether(FragmentManager.java:2411)
  at android.support.v4.app.FragmentManagerImpl.removeRedundantOperationsAndExecute(FragmentManager.java:2366)
  at android.support.v4.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:2273)
  at android.support.v4.app.FragmentManagerImpl$1.run(FragmentManager.java:733)
  at android.os.Handler.handleCallback(Handler.java:873)
  at android.os.Handler.dispatchMessage(Handler.java:99)
  at android.os.Looper.loop(Looper.java:214)
  at android.app.ActivityThread.main(ActivityThread.java:7037)
  at java.lang.reflect.Method.invoke(Method.java:-2)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:494)
  at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:965)
```

**Comment by semanticer** [comment]
Can't reproduce this one.  It looks like we are adding MainFragment but if we look at events it really seems to be connected to ReportsSuccess “ShortcutToday”

**Comment by kornelijepetak** [comment]
> Can't reproduce this one. It looks like we are adding MainFragment but if we look at events it really seems to be connected to ReportsSuccess “ShortcutToday”

You should be aware that the list of events you can see on AppCenter is what executed (most often without errors) before the crash, not (necessarily) what caused the crash. Be very careful when interpreting the crash by looking at the events.

**Comment by semanticer** [comment]
I updated the impact from our 2.4.1 larger dataset. This is the most prominent issue on 2.4.1. I don't know how but we should try to fix this in 2.6. What do you think @kornelijepetak ?

**Comment by kornelijepetak** [comment]
@semanticer Unfortunatelly, this was not fixed in 2.6.
It is visible as a crash in 2.6 every once in a while tho.
It is also currently the most prominent crash in the release, so it should be taken care of in the near future.

**Comment by semanticer** [comment]
Oh, I knew it will end up in my release again :) I'm moving this to 2.8 @amulware ➿
